### PR TITLE
:man_technologist: Changes in LoRaMAC.cpp Class C AU 915

### PR DESCRIPTION
**Pull request by aleamar264**
Fix Class C AU 915

### CODE REVIEW

**Review by sabas1080 — APPROVED** [review]

File: src/arduino-rfm/LoRaMAC.cpp [hunk 1]
```diff
@@ -39,6 +39,10 @@
 #include "Config.h"
 #include "Arduino.h"
 
+#define AU_915 // use the band youy need
```
Comment (wero1414) [inline src/arduino-rfm/LoRaMAC.cpp:42]: The library specifies that this needs to be defined on the file Config.h not in the LoRaMAC.cpp

File: src/arduino-rfm/LoRaMAC.cpp [hunk 2]
```diff
@@ -39,6 +39,10 @@
 #include "Config.h"
 #include "Arduino.h"
 
+#define AU_915 // use the band youy need
+#define SUBND_0 // same here!
```
Comment (wero1414) [inline src/arduino-rfm/LoRaMAC.cpp:43]: This should be defined con the config.h too

**Review by wero1414 — CHANGES_REQUESTED** [review]
Need to clean the defines on the file LoRaMAC.cpp the subband config seems good

**Comment by sabas1080** [comment]
Now available in #138

**Pull request closed without merging**